src/index.ts

```diff
@@ -563,8 +563,18 @@ export const [isAirplaneMode, isAirplaneModeSync] = getSupportedPlatformInfoFunc
   defaultValue: false,
 });
 
-export const getDeviceType = () => {
-  if (Platform.OS === 'windows') return 'Desktop';
+export const deviceType = {
+  DESKTOP: 'Desktop',
+  HANDSET: 'Handset',
+  TABLET: 'Tablet',
+  TV: 'Tv',
+  UNKNOWN: 'unknown'
+} as const
+
+type DeviceTypeReturnType = typeof deviceType[keyof typeof deviceType]
```

Comment:
Try something like this:

```ts
export enum DeviceTypeReturnType {
    DESKTOP = 'Desktop',
    HANDSET = 'Handset',
    TABLET = 'Tablet',
    TV = 'Tv',
    UNKNOWN=  'unknown'
}
```

personally, it's a lot cleaner / easier to understand

Reply:
Oh my - I had not realized that this had gone stale, my apologies @chrisdrackett - I agree in general with @schie's comment, and just wanted to add that index.ts has a conflict now but it should be easy to resolve, I think it's just the addition of windows support for device type, and the new possible 'GamingConsole' return value (which should be added to this PR's set of possible values) from #1238

Reply:
I totally forgot about this, I'm all for adding an enum as well, so hopefully I can find time to update this if someone else does not get to it first (we're in the middle of a pretty big launch so things like this aren't on my radar atm.)

Reply:
@mikehardy would you consider this a breaking change?

Reply:
I think "no" because, I believe you can still compare the strings

Reply:
@schie I could only definitively say with testing, sorry - while I basically insist on using typescript where ever I can, I'm not an expert (yet?)

Reply:
this should not be a breaking change as outside of TS these are still the same strings

Reply:
if someone's typescript build breaks, it's a breaking change
(note: breaking changes aren't evil, we can release them no problem, we just want to make sure the version increment  + changelog correctly communicates the changes developers need to look out for)